packages/grid/src/grid.tsx

```diff
@@ -17,11 +17,9 @@ export function Grid( {
 	editMode = false,
 	onChangeLayout,
 }: GridProps ) {
-	// Temporary layout to avoid updaing the layout while dragging
-	const [ temporaryLayout, setTemporaryLayout ] = useState< GridLayoutItem[] | undefined >(
-		layout
-	);
-	const activeLayout = temporaryLayout || layout;
+	// Temporary layout to avoid updating the layout while dragging
+	const [ temporaryLayout, setTemporaryLayout ] = useState< GridLayoutItem[] | undefined >();
+	const activeLayout = temporaryLayout ?? layout;
```

Comment:
This make sense, this means that we continue to use the prop value until the user makes a change. Now what this suggests as well is that we should do `setTemporaryLayout( undefined )` whenever we call `onChangeLayout` to avoid issues afterwards.

Reply:
I was thinking the opposite calling `setTemporaryLayout( undefined )` at the same time we call `onChangeLayout` instead of when the layout changes externally. 

I don't recall exactly why we use `temporaryLayout` for but if it's for user edits or things like that, I think these would need to have more priority than external changes (when these user changes are not persisted, and they get persisted when you call `onChangeLayout`).